source/common/network/tcp_listener_impl.cc

```diff
@@ -90,18 +91,25 @@ void TcpListenerImpl::onSocketEvent(short flags) {
     cb_.onAccept(
         std::make_unique<AcceptedSocketImpl>(std::move(io_handle), local_address, remote_address));
   }
+
+  ENVOY_LOG_MISC(trace, "TcpListener {} accepted {} new connections.",
```

Comment:
Should we add a histogram of this? It's hard to know how to tune the knob without knowing the distribution of this.